stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/factory/AttachmentFactory.kt

```diff
@@ -0,0 +1,36 @@
+package io.getstream.chat.android.compose.ui.attachments.factory
```

Comment:
Would we have to introduce this import change if we moved the `AttachmentFactory` in the same package level as the `StreamAttachmentFactory` file?

@zsmb13 @bychkovdmitry do you think it would make sense to put it there?

Reply:
Yeah, I'd vote for keeping it in its current package - fewer breaking changes for existing usages (fewer articles, tutorials, etc we have to update too)

Reply:
Yes, if it is big breaking change (articles, tutorials, etc.), then I'm rolling it back.